go-controller/pkg/util/nicstobridge.go

```diff
@@ -218,11 +219,17 @@ func NicToBridge(iface string) (string, error) {
 	setupDefaultFile()
 
 	// Get ip addresses and routes before any real operations.
-	addrs, err := netlink.AddrList(ifaceLink, syscall.AF_INET)
+	family := syscall.AF_INET
+	if config.IPv6Mode && !config.IPv4Mode {
+		family = syscall.AF_INET6
+	} else if config.IPv6Mode && config.IPv4Mode {
+		family = syscall.AF_UNSPEC
```

Comment:
Actually, we should just use `AF_UNSPEC` here regardless of config. We are moving the entire interface onto the bridge. Even if ovn-kubernetes is running IPv4-only, if `eth0` has an IPv6 address, we should move that IPv6 address onto the bridge, or else it becomes inaccessible.

(Likewise in BridgeToNic)